spec/features/user_views_all_books_spec.rb

```diff
@@ -7,8 +7,8 @@
 
     visit books_path
 
-    expect(page).to have_content("Practical Vim")
-    expect(page).to have_content("The Giver")
+    expect(page).to have_book("Practical Vim")
+    expect(page).to have_book("The Giver")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.